meta-mbl-distro/scripts/create-partition-table-markdown

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import pathlib
+import sys
+
+# Set the PYTHONPATH so we can import openembedded scripts/lib modules.
+oe_scripts_path = (
+    pathlib.Path(__file__).resolve().parent.parent.parent.parent
+    / "openembedded-core"
+    / "scripts"
+)
+
+lib_path = oe_scripts_path / "lib"
+sys.path.insert(0, str(lib_path))
+
+import scriptpath
+import scriptutils
+
+# This is needed so we can import bitbake modules.
+bitbakepath = scriptpath.add_bitbake_lib_path()
+
+import bb.tinfoil
+
+import decimal
```

Comment:
Standard library imports should be above third party imports.

Reply:
`import decimal` removed in https://github.com/ARMmbed/meta-mbl/pull/698/commits/2968ed0dfa47259fe02438569650379f7fa15947